frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java

```diff
@@ -345,22 +346,46 @@ private void handleScaleModel(
             ChannelHandlerContext ctx, QueryStringDecoder decoder, String modelName) {
         int minWorkers = NettyUtils.getIntParameter(decoder, "min_worker", 1);
         int maxWorkers = NettyUtils.getIntParameter(decoder, "max_worker", 1);
+        String mode = NettyUtils.getParameter(decoder, "mode", null);
```

Comment:
Why not have a boolean with "sync"? Can we have anything other than "sync" and "async"?

Reply:
True, but keeping mode in a string feels more descriptive, We have two parameter methods, for key value pairs int and String, either way we do not get a boolean directly, even if the customer can pass something like sync=1 , does it prevent them from having a value more than 1?